hack/build-images.sh

```diff
@@ -56,8 +62,9 @@ function image {
   docker tag -f $1:latest $1:${OS_RELEASE_COMMIT}
 }
 
-# images that depend on scratch
+# images that depend on scratch / centos
 image openshift/origin-pod                   images/pod
+image openshift/openvswitch                  images/openvswitch
```

Comment:
No, it's just centos7 + openvswitch. However nothing builds on top of it as the node image inherits from the origin image instead.